Sources/Plasma/FeatureLib/pfDXPipeline/plDXPipeline.cpp

```diff
@@ -10584,8 +10591,8 @@ struct plShadowVertStruct
 void plDXPipeline::IReleaseBlurVBuffers()
 {
     const uint32_t kVSize = sizeof(plShadowVertStruct);
-    int i;
-    for( i = 0; i < kMaxRenderTargetNext; i++ )
+
+    for(int i = 0; i < kMaxRenderTargetNext; i++ )
```

Comment:
I wouldn't worry too much about these statements. The possibility of confusion of the reader is low - and this sort of churn is kind of an issue generally for people who want to rebase. But that's just my opinion. Generally it feels like H'uru doesn't take on really low priority code style issues to reduce churn. But that's just my opinion.